### PR TITLE
Additional settings

### PR DESCRIPTION
**Pull request by atltt**
It's pretty good

### CODE REVIEW

**Comment by robobenklein** [comment]
resolves #30

**Comment by atltt** [comment]
On it Ben

File: package.json [hunk 1]
```diff
@@ -80,6 +80,60 @@
...
+    "default_git_remote": {
+      "title":  "Default Git Remote",
+      "description": "Changes the Git remote",
+      "type": "string",
+      "default": "origin",
+      "order": 45
+    },
+    "default_drawer_height": {
+      "title": "Drawer Height",
+      "description": "Change the Drawer: Either the number of lines OR pixels height",
```
Comment (robobenklein) [inline package.json]: No I mean *you should choose* one, not both XD

Comment (robobenklein) [reply package.json]: For what I mean by drawer, look at the second pic in #9

Comment (robobenklein) [reply package.json]: And also note that the drawer is resizable after it's been opened still

**Comment by robobenklein** [comment]
image

still need to set regex as string, for now use #33's example regex as the default

also let's make the default height 12 lines, 64 lines would take up most of the screen

**Comment by atltt** [comment]
lol oof

**Comment by atltt** [comment]
thank the Lord

**Pull request merged**